assets/js/modules/reader-revenue-manager/index.js

```diff
@@ -71,3 +79,35 @@ export const registerModule = ( modules ) => {
 		},
 	} );
 };
+
+export const registerNotifications = ( notifications ) => {
+	notifications.registerNotification( 'setup-success-notification-rrm', {
+		Component: RRMSetupSuccessSubtleNotification,
+		priority: 10,
+		areaSlug: NOTIFICATION_AREAS.BANNERS_BELOW_NAV,
+		viewContexts: [
+			VIEW_CONTEXT_MAIN_DASHBOARD,
+			VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY,
```

Comment:
While we're at it, we can prevent the notification from appearing on the view only dashboard.

```suggestion
```